content/blog/2022/2022-08-25-schedule-cron-job-in-node.md

```diff
@@ -0,0 +1,318 @@
+lay dormant until it is required. Another example of a deamon is the web server. The web server remains idle until a request for a web page is made. 
+
+## Using Node Cron in Node.Js
+In this article, we'll use the [node-cron](https://www.npmjs.com/package/node-cron) library to schedule cron jobs in node.js and build several demo applications. Node-cron is a node module that is made available by npm.
+
+To schedule jobs using `node-cron`, you need to invoke the cron `.schedule` method.
+
+### Cron Schedule Method Syntax
+```
+cron.schedule(expression , function, options);
+```
+As shown above, the cron `.schedule` method takes in three arguments where :
+
+#### The First Argument (Expression)
+This is a cron expression also known as crontab. This expression is used to specify when a cron job is to be executed. The cron expression used here can be represented using Asterix (*),  which are wildcard characters representing the scheduled time for cron jobs. 
+
+The cron expression by default will be in the format `* * * * * *`.
+
+Here's a quick reference to the cron expression, Indicating what each asterisks represents.
+```
+ # ┌────────────── second (0 - 59) (optional)
+ # │ ┌──────────── minute (0 - 59) 
+ # │ │ ┌────────── hour (0 - 23)
+ # │ │ │ ┌──────── day of the month (1 - 31)
+ # │ │ │ │ ┌────── month (1 - 12)
+ # │ │ │ │ │ ┌──── day of the week (0 - 6) (0 and 7 both represent Sunday)
+ # │ │ │ │ │ │
+ # │ │ │ │ │ │
+ # *  * * * * *  command to execute
+```
+You can replace each asterisk with an appropriate number (or character) such that the expression describes the time the job will be executed.
+
+Cron expression has some special characters like:
+- `*` Asterisk operator this is a wildcard and acts as a placeholder. It means a value or always. Eg. If the asterisk symbol is in the month field, it means performing these specified tasks every month.
+- `,` The comma operator allows you to specify a list of values for repetition. Eg. If you have `1, 3, 5` in the `month field`, the task will run on days of `month 1, 3, and 5`.
+- `-` The hyphen operator allows you to specify a range of values. Eg. If you have 1-5 in the `Day of the week field`, the task will run every weekday (From Monday to Friday).
+- `/` The slash operator allows you to specify values that will be repeated over a certain interval between them. Eg. if you have */4 in the Hour field, It means the action will be performed every 4 hours.
+
+Each field can have one or more values separated by commas, or a range of values separated by hyphens.
+
+If you're unsure about manually writing cron syntax, you can, use free tools like [Crontab Generator](https://crontab-generator.org/) or [Crontab.guru](https://crontab.guru/#*_*_*_*_*) to generate the exact time date you want for your cron expression.
+
+####  The Second Argument (Function)
```

Comment:
Same here, the header shouldn't be "The Second Argument", but rather "The Cron Function" (or similar).